packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -142,7 +142,7 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   private _getAddNewElement(): JSX.Element {
     let { personaSize } = this.props;
     return <button { ...getNativeProps(this.props.addButtonProps, buttonProperties) }
-      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton') }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton', styles.addButton, styles.itemButton) }>
```

Comment:
I don't really understand the intent here; why button is used and not Persona itself?